### PR TITLE
quick easy fix to give a bit more room to display amounts the fill th…

### PR DESCRIPTION
**Pull request by ScottDowne**
…e buttons

### CODE REVIEW

File: less/shared.less [hunk 1]
```diff
@@ -273,7 +273,7 @@ input[name="donation_amount"] + label,
 input[name="payment-type"] + label {
   display: inline-block;
   width: 100%;
-  padding: 6px 12px;
+  padding: 6px 0;
```
Comment (jbuck) [inline less/shared.less]: This also affects the placement of the $ symbol in the last box

**Comment by jbuck** [comment]
r-

**Comment by ScottDowne** [comment]
I cleaned up some of the css class naming so I could more specifically target the thing I wanted in a clear way.

**Comment by ScottDowne** [comment]
@jbuck r?

**Comment by jbuck** [comment]
@ScottDowne r+

**Pull request merged**